src/components/reviews/reviews.tsx

```diff
@@ -1,76 +1,86 @@
+        {sortedReviews.map(({ id, date, user, comment, rating }) => {
           const reviewDate = dayjs(date).format(DATE);
           const year = dayjs(date).year();
           const month = dayjs(date).format(MONTH_TEXT);
 
-          if (index < MAX_REVIEWS) {
-            return (
-              <li className="reviews__item" key={id}>
-                <div className="reviews__user user">
-                  <div className="reviews__avatar-wrapper user__avatar-wrapper">
-                    <img
-                      className="reviews__avatar user__avatar"
-                      src={user.avatarUrl}
-                      width={54}
-                      height={54}
-                      alt="Reviews avatar"
-                    />
-                  </div>
-                  <span className="reviews__user-name">{user.name}</span>
+          return (
+            <li className="reviews__item" key={id}>
+              <div className="reviews__user user">
+                <div className="reviews__avatar-wrapper user__avatar-wrapper">
+                  <img
+                    className="reviews__avatar user__avatar"
+                    src={user.avatarUrl}
+                    width={54}
+                    height={54}
+                    alt="Reviews avatar"
+                  />
                 </div>
-                <div className="reviews__info">
-                  <div className="reviews__rating rating">
-                    <div className="reviews__stars rating__stars">
-                      <span
-                        style={{
-                          width: `${rating * ONE_STAR_RATIO}%`,
-                        }}
-                      />
-                      <span className="visually-hidden">Rating</span>
-                    </div>
+                <span className="reviews__user-name">{user.name}</span>
+              </div>
+              <div className="reviews__info">
+                <div className="reviews__rating rating">
+                  <div className="reviews__stars rating__stars">
+                    <span
+                      style={{
+                        width: `${rating * ONE_STAR_RATIO}%`,
+                      }}
+                    />
+                    <span className="visually-hidden">Rating</span>
                   </div>
-                  <p className="reviews__text">{comment}</p>
-                  <time className="reviews__time" dateTime={reviewDate}>
-                    {month} {year}
-                  </time>
                 </div>
-              </li>
-            );
-          }
-
-          return null;
+                <p className="reviews__text">{comment}</p>
+                <time className="reviews__time" dateTime={reviewDate}>
+                  {month} {year}
+                </time>
+              </div>
+            </li>
+          );
         })}
       </ul>
-      {isAuth && <ReviewForm />}
+      {isAuth && <ReviewForm offerId={offerId} />}
```

Comment:
```suggestion
      {isAuth && <ReviewForm />}
```

Можно не прокидывать проп `offerId`?